deploy/helm/charts/values.yaml

```diff
@@ -75,13 +75,14 @@ serviceAccount:
   name:
 
 deviceClass:
-  # Defines name of storage class.
+  # Name of default device StorageClass.
   name: openebs-device
+  # If true, enables creation of the openebs-device StorageClass
+  enabled: true
   # Available reclaim policies: Delete/Retain, defaults: Delete.
   reclaimPolicy: Delete
-  # set as default class
+  # If true, sets the openebs-device StorageClass as the default StorageClass
   isDefaultClass: false
-  storageType: "device"
```

Comment:
This value is still used in the storage class template

Reply:
Could you elaborate on the comment? I wasn't able to understand.

Reply:
This value is used [here](https://github.com/openebs/dynamic-localpv-provisioner/pull/46/files#diff-7413840c81712c7ded1444ef89aadcebdbee7d7d4414e252b9c63155cd550df9R11). So either it should be changed at both places or replaced with something else.

Reply:
I'm sorry but I still wasn't able to understand properly. Can we please jump into a Google Meet for 10 min?

Reply:
Thanks for the meeting. I got the context. Removed StorageType configuration capabilities from device-class.yaml and hostpath-class.yaml. Good catch. Thanks.